actions/hadoop-upgrade

```diff
@@ -0,0 +1,253 @@
+#!/bin/bash
+action-set charmdir.before=${CHARM_DIR}
```

Comment:
Perhaps you're right but this brings up a bigger question: 
https://github.com/andrewdmcleod/layer-hadoop-base/blob/upgrade/actions/hadoop-upgrade#L209
If the fetch fails this script should exit anyway - why didn't it for you?